.github/workflows/liveness-branch-sync.yml

```diff
@@ -15,6 +15,6 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
       - name: Create sync pull request
-        run: gh pr create -B liveness-release -H main --title 'Sync branch main to liveness-release' --body 'Created by Github action'
+        run: length=$(gh pr list -B liveness-release -H main -s open --json title --jq "length"); if [[ "$length" -gt 0 ]]; then echo "PR already exists"; true; else false; fi|| gh pr create -B liveness-release -H main --title 'Sync branch main to liveness-release' --body 'Created by Github action'
```

Comment:
Can we put this in a multiline and prettify it for readability purposes?

Also, don't think those `true` and `false` are necessary and we can just do

```
if [[ "$length" -gt 0 ]]; then
    #... 
else
    # ...
fi
```
unless I'm missing something.

Reply:
ah was just doing that to keep it one line, but can change it :)

Reply:
yep, let's do it!